src/drools/ruleset.py

```diff
@@ -248,7 +250,25 @@ def get_by_session_id(cls, session_id: int) -> Ruleset:
         )
 
 
+message_counter = 0
+java_lang_System = None
+
+
+def call_garbage_collector():
+    global message_counter, java_lang_System
+    if message_counter > DROOLS_JPY_GC_AFTER:
+        if java_lang_System is None:
+            import jpy
+
+            java_lang_System = jpy.get_type("java.lang.System")
+        java_lang_System.gc()
```

Comment:
@benthomasson Should this be conditional on DROOLS_JPY_GC_AFTER > 0?

Reply:
Sure